requirements.txt

```diff
@@ -0,0 +1,53 @@
+aiohttp==3.9.5
```

Comment:
For python projects it's a good practice to freeze dependencies so that when you restore the project, it uses the same dependencies as intended